docs/lcb/notes/22379.md

```diff
@@ -0,0 +1 @@
+# [22379] Fix error calling foreign handlers with non-default C call type on 64-bit Windows
```

Comment:
Fix error calling foreign handlers with non-default calling convention on 64-bit Windows

Reply:
I've edited the comments now